### PR TITLE
Add more customization options to the Field component

### PR DESCRIPTION
**Pull request by WebCloud**
Add option to customize the `border-shadow` when focused and also enhance the inline icon colouring.

### CODE REVIEW

File: Field/index.jsx [hunk 1]
```diff
@@ -225,7 +225,7 @@ const Field = React.createClass({
           borderColor: this.state.hover || focus
             ? customize.borderColorSelected
             : customize.borderColor,
-          boxShadow: focus && `0 0 4px ${customize.borderColorSelected}`
+          boxShadow: focus && (customize.boxShadow ? customize.boxShadow : `0 0 4px ${customize.borderColorSelected}`)
```
Comment (xaviervia) [inline Field/index.jsx]: Where is `customize.boxShadow` coming from? In the `withTheme` HoC is not being set

Comment (WebCloud) [inline Field/index.jsx]: I've added directly there. I guess I should add to the HoC instead...

Comment (xaviervia) [inline Field/index.jsx]: Probably both :D

Comment (WebCloud) [inline Field/index.jsx]: Just for future reference, as discussed in person, these changes are ok to be carried as this for those customizations won't be officially supported by the Theme/Merchant Theming features.

**Pull request merged**